src/ducks/balance/Balance.jsx

```diff
@@ -398,6 +406,14 @@ class Balance extends PureComponent {
             nbAccounts: accounts.length
           }
 
+    const allOwners = flattenDeep(
+      accounts.map(account => get(account, 'owners.data', []))
+    )
```

Comment:
Why does normal flatten does not work ? I am not very familiar with the owners.data structure. 

Nit: Maybe this could be extracted to a more descriptive helper ?

Reply:
Flatten is enough. I extracted it.